### PR TITLE
KRV-673 : Implement Gounity code change to enable tenant 

### PR DESCRIPTION
**Pull request by karthikk92**
refer confluence : https://confluence.cec.lab.emc.com/display/CSIUNITY/Multi+Tenancy+with+Unity+NFS

installation of driver:

Tenant added

values.yaml detail:

### CODE REVIEW

**Review by harishp8889 — DISMISSED** [review]

**Review by rajendraindukuri — APPROVED** [review]

File: types/requests.go [hunk 1]
```diff
@@ -53,6 +53,11 @@ type LunCreateParam struct {
 	LunParameters *LunParameters `json:"lunParameters"`
 }
 
+// Struct to capture the Tenants
+type Tenants struct {
+	TenantId string `json:"id"`
```
Comment (rensyct) [inline types/requests.go:58]: TenantId -> TenantID

Comment (rensyct) [inline types/requests.go:58]: Please take care of the indentation

File: types/requests.go [hunk 2]
```diff
@@ -169,6 +174,8 @@ type HostCreateParam struct {
 	Name        string `json:"name"`
 	Description string `json:"description"`
 	OsType      string `json:"osType"`
+	Tenant *Tenants `json:"tenant,omitempty"`
```
Comment (rensyct) [inline types/requests.go:177]: Please take care of the indentation

**Review by rensyct — APPROVED** [review]

File: types/requests.go (hunk 1 above)
Comment (karthikk92) [reply types/requests.go]: taken care

Comment (karthikk92) [reply types/requests.go]: will be taken care in next pr

File: types/requests.go (hunk 2 above)
Comment (karthikk92) [reply types/requests.go]: taken care

**Review by rensyct — APPROVED** [review]

**Review by prasar6 — APPROVED** [review]
/lgtm

**Pull request merged**